
function __error()
    print("sys error")
end
function test()
    print("sys error")
end

function main()
    local c = 123
    local b = 456;
    print(c+b)
    print("main function")
end
